executor.go

```diff
@@ -629,34 +626,20 @@ func localExecute(pipeline *zjson.Pipeline, id int64, executionId string, cfg Co
 		return
 	}
 
-	executionFiles := s3key
-
 	eg, egCtx := errgroup.WithContext(ctx)
 	eg.SetLimit(runtime.NumCPU())
 
-	uploadedFiles := []string{}
 	for path, image := range blocks {
+		path := path
+		image := image
```

Comment:
What's the purpose of reassigning those values

Reply:
https://pkg.go.dev/golang.org/x/tools/go/analysis/passes/loopclosure